funders/czi.md

```diff
@@ -0,0 +1,18 @@
+# [`Chan Zuckerberg Initiative`](https://chanzuckerberg.com/eoss/)
+
+## About
+
+Open source software is crucial to modern scientific research, advancing biology and medicine while providing reproducibility and transparency. Yet even the most widely-used research software often lacks dedicated funding. CZI’s Essential Open Source Software for Science program supports software maintenance, growth, development, and community engagement for critical open source tools.
+
+## Eligibility Criteria
+
+We welcome applications from any country, provided the proposed work is compliant with the United States Treasury Department’s Office of Foreign Asset Control (OFAC) sanctions program.
+Applications may be submitted by non-profit and for-profit organizations, public and private institutions, such as colleges, universities, hospitals, laboratories, units of state and local government, companies, and eligible agencies of the federal government. As part of the full application process, for-profit organizations may need to provide additional information on the charitable purposes of the proposal. Grants are not permitted to individuals; only to organizations. Open source software projects operating independently must be affiliated with an organization, as described below.
+
+## Application
+
+<https://chanzuckerberg.com/rfa/essential-open-source-software-for-science/>
+
+## Deadline
+
+Cycle 6 closed in October 2023.
```

Comment:
Is this still open, then?

Reply:
No but they do run future cycles so this date may indicate the next funding cycle I suppose. (This is copied from the open-funders repo)

Reply:
No it is closed. I think the date is there to give indication to the funding cycle.